bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/scheduler/ThreadPoolManager.java

```diff
@@ -5,16 +5,25 @@
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-v10.html
  */
-package org.eclipse.smarthome.core.common;
+package org.eclipse.smarthome.core.scheduler;
```

Comment:
I am open for anything basically. I will only have 1 problem in the coming week(s) and that is a lack of time, so I am open for you/someone else to take over the PR to get it finished.